### PR TITLE
Show setup deposits task for PO accounts without sales and right after account creation

### PR DESCRIPTION
**Pull request by oaratovskyi**
#### Changes proposed in this Pull Request

* Show setup deposits task for PO accounts without sales and right after acount creation

#### Testing instructions

* Checkout `fix/show-connect-deposits-for-po-accounts-without-sales-before-14-days`
* Onboard new PO account
* Check that you see the connect deposits task
    image
* Click on Start Receiving Deposits, complete the remaining steps of the PO
* Check that account is complete now (may be in pending verification for a min)
    image

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _'QA Testing Not Applicable'_
- [x] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [x] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `6433` or branch name `fix/show-connect-deposits-for-po-accounts-without-sales-before-14-days` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=fix/show-connect-deposits-for-po-accounts-without-sales-before-14-days) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: a9ec5296583ac3d0ffc7394a54e4a24ed8ec9303
- Build time: 2023-06-02 15:11:10 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: client/overview/task-list/po-tasks.tsx [hunk 1]
```diff
@@ -41,7 +41,12 @@ export const getVerifyBankAccountTask = (): any => {
 
 		// when account is created less than 14 days ago, no need to show any notice
```
Comment (ismaeldcom) [inline client/overview/task-list/po-tasks.tsx]: We should probably also update the comment 🙂

**Review by ismaeldcom — APPROVED** [review]
Tested using a JN site and it works as expected! Left a nitpick, but preapproving it! 🚀

File: client/overview/task-list/po-tasks.tsx (hunk 1 above)
Comment (oaratovskyi) [reply client/overview/task-list/po-tasks.tsx]: fixed in https://github.com/Automattic/woocommerce-payments/pull/6433/commits/a9ec5296583ac3d0ffc7394a54e4a24ed8ec9303, thanks!

**Pull request merged**